tools/trustee/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use log::{debug, error};
+
+mod cli;
+mod keys_certs;
+
+#[cfg(feature = "aliases")]
+mod aliases;
+
+#[cfg(feature = "plugins")]
+mod plugins;
+
+#[tokio::main]
+async fn main() {
+    env_logger::init_from_env(env_logger::Env::new().default_filter_or("info"));
+
+    #[cfg(feature = "aliases")]
+    match aliases::match_alias() {
+        Ok(_) => return,
```

Comment:
Maybe I don't understand the aliases feature at all but why would we return here?